go.mod

```diff
@@ -136,5 +137,6 @@ require (
 
 replace (
 	github.com/denisenkom/go-mssqldb v0.0.0-20181014144952-4e0d7dc8888f => github.com/denisenkom/go-mssqldb v0.0.0-20161128230840-e32ca5036449
+	github.com/go-macaron/cors v0.0.0-20190309005821-1733aeedd68a => github.com/tamalsaha/cors v0.0.0-20190309005821-1733aeedd68a
```

Comment:
Why is this replace needed? https://github.com/go-macaron/cors is up-todate with github.com/tamalsaha/cors

Reply:
See: https://github.com/go-macaron/cors/pull/1
it fails otherwise

Reply:
@techknowlogick , I just updated https://github.com/go-macaron/cors/commit/6fd6a9bfe14e9ed6ddf9265b6580f7638105b27b . Please revendor . This should be fixed.

Reply:
Thanks for fix 😄